src/applications/letters/tests/actions/letters.unit.spec.js

```diff
@@ -26,11 +24,6 @@ import {
   getLetterPdf,
 } from '../../actions/letters';
 
-Sentry.init({
```

Comment:
This Sentry testkit setup has been moved to `mocha-setup.js` for use ease of use in other unit tests.

**Note:** Because we're still running Mocha v5, we can't use global hooks to `testkit.reset()` after each test, since global hooks are introduced in Mocha v8, so I left that alone in this module, and we'll need to add that any time we use the testkit.